tests/Microsoft.Identity.Web.Test/WebAppExtensionsTests.cs

```diff
@@ -877,17 +877,19 @@ public void PreventChangesInOpenIdConnectOptionsToBeOverlooked()
         {
             // If the number of public properties of OpenIdConnectOptions changes,
             // then, the PopulateOpenIdOptionsFromMergedOptions method
-            // needs to be updated. For this uncomment the 2 lines below, and run the test
+            // needs to be updated. For this uncomment the System.IO.File.WriteAllLines() lines below, and run the test
             // then diff the files to find what are the new properties
             int numberOfProperties = typeof(OpenIdConnectOptions).GetProperties(System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Instance).Length;
 #if NET9_0_OR_GREATER
             int expectedNumberOfProperties = 61;
+            //System.IO.File.WriteAllLines(@"C:\temp\net9.txt", typeof(OpenIdConnectOptions).GetProperties().Select(p => p.Name));
```

Comment:
nit: save file path to a variable here and call `WriteAllLines` after `#endif` to reduce code duplication.

Reply:
but all the file paths are different, so how sure how a variable name will help in this case?

Reply:
A little cleaner, I think:
```csharp
            int numberOfProperties = typeof(OpenIdConnectOptions).GetProperties(System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Instance).Length;
#if NET9_0_OR_GREATER
            int expectedNumberOfProperties = 61;
            string filePath = @"C:\temp\net9.txt";
#elif NET8_0
            int expectedNumberOfProperties = 60;
            string filePath = @"C:\temp\net8.txt";
#else
            int expectedNumberOfProperties = 57;
            string filePath = @"C:\temp\net7Below.txt";
#endif
            //System.IO.File.WriteAllLines(filePath, typeof(OpenIdConnectOptions).GetProperties().Select(p => p.Name));
            Assert.Equal(expectedNumberOfProperties, numberOfProperties);
        }
```

Reply:
I see, thanks for clarifying.

Reply:
Thanks @pmaytak  & @jennyf19 !